src/components/Chef.js

```diff
@@ -0,0 +1,96 @@
+import { Component, PropTypes, DOM as dom, createFactory } from 'react';
+import { connect } from 'react-redux';
+import { Link as LinkComponent } from 'react-router';
+import actions from '../action_creators';
+import constants from '../constants';
+
+const Link = createFactory(LinkComponent);
+const displayMax = 6;
+
+function mapStateToProps(state) {
+  return {
+    orders: state.orders
+  };
+}
+
+function toOpenEntries(orderId, entry, entryIndex) {
+  return {
+    orderId,
+    entry,
+    entryIndex
+  };
+}
+
+class Chef extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.cancel = this.cancel.bind(this);
+    this.state = {
+      openEntries: [],
+      selectedEntry: undefined
+    };
+  }
+
+  setOpenEntries(orders) {
```

Comment:
So have been thinking some more about this.. I wonder if we should do it the way we did it in `Orders` with `getFilteredOrders`. We could have a method `getOpenEntries` that would filter and return the open entries using `this.props.orders`. Then, in `render`, we could just call `this.getOpenEntries().map...` to render them. That way, we won't have to manage this extra local state `openEntries`, and we wouldn't need `componentWillReceiveProps` at all.